hera_cal/vis_clean.py

```diff
@@ -335,9 +405,29 @@ def vis_clean(self, keys=None, x=None, data=None, flags=None, wgts=None,
             time. Skipped channels are then flagged in self.flags.
             Only works properly when all weights are all between 0 and 1.
         verbose : Lots of outputs
+        tol : float, optional. To what level are foregrounds subtracted.
         overwrite : bool, if True, overwrite output modules with the same name
                     if they already exist.
-        tol : float, optional. To what level are foregrounds subtracted.
+        skip_flagged_edge_freqs : bool, optional
+            if true, do not filter over flagged edge frequencies (filter over sub-region)
+            defualt is False
+        skip_flagged_edge_times : bool, optional
+            if true, do not filter over flagged edge times (filter over sub-region)
+            defualt is False
+        skip_gaps_larger_then_filter_period : bool, optional
+            if true, skip integrations or channels with gaps that are larger then the period of
+            of the finest scale mode used for interpolation.
+            default is False.
+        clean_flags_in_resid_flags : bool, optional
+            if true, include clean flags in residual flags that will be written out in res_outfilename.
+            default is False.
+        flag_filled : bool, optional
```

Comment:
Hmm, can you change this parameter name to something a little more informative, perhaps `keep_flags`
and also the term `filter flags` is a little ambiguous, that makes me thing of the flags used for performing the CLEAN. Perhaps say `set the post-filtered flags equal to ...`

Reply:
I've adopted this language and gotten rid of the xtalk analogy.